membership_file/util.py

```diff
@@ -58,5 +60,5 @@ def dispatch(self, request, *args, **kwargs):
             return HttpResponseRedirect(resolve_url(self.fail_url))
         if not request.member.is_considered_member() and self.requires_active_membership:
             # Current session has a disabled member connected
-            return HttpResponseRedirect(resolve_url(self.fail_url))
+            raise UserIsNotCurrentMember()
```

Comment:
Why isn’t the first line changed, but the second one is?

Reply:
Because you have eyes and I don't 😎